spec.md

```diff
@@ -71,11 +71,11 @@ system. This is most typically when that system receives an external signal
 
 #### Event
 Data representing an occurrence, a change in state, that something happened
-(or did not happen), usually used for notification. Events include context
-and data. Each occurrence might be uniquely identified with data in the event.
-Events are considered as facts that have no given destination, whereas
-messages contain intent and tend to transport data from a source to a given
-destination.
+(or did not happen).  Events include context and data.  Each occurrence MAY be
+uniquely identified with data in the event. Events ought not be confused with
+messages which are used to carry intent and data to a given destination. Events
```

Comment:
It's a good idea to explain the difference between messages and events, but I think messages are usually considered as containers to transport or distribute data between systems without specifying the semantic of the data.
I would see them in a layer below events. Usually on the same layer, events are distinguished from commands. Commands carry an intent about something that is supposed to happen on the target. Events inform about something that happened at the source. Both can be transported as messages.

Reply:
That is exactly what was going on in my head when I wrote this. But I just wanted to disambiguate what was already there. I thought if I added a new terminology i.e. `command`, not only would it be very specific to an architectural guideline, it may cause further confusion.

Reply:
I get your point. That's the problem with terms like messages and events. They have quite some history in software engineering. I mentioned commands to have some distinction on the same layer, but I understand if you don't want to introduce yet another term.
I just think that messages will play an important role in this spec, as protocols like AMQP are mentioned as "transport protocols" for events. 
What about: 
```
Events ought not to be confused with messages which are used to transport or distribute data without assumptions regarding its semantic.
```

Reply:
That looks good. I have changed my commit to reflect this.